pom.xml

```diff
@@ -53,8 +53,8 @@
 
         <!-- Build -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.source.version>1.7</java.source.version>
-        <java.target.version>1.7</java.target.version>
+        <java.source.version>1.8</java.source.version>
+        <java.target.version>1.8</java.target.version>
```

Comment:
By global search `java.source.version` in project, `sentinel-api-gateway-adapter-common`, `sentinel-cluster`, `sentinel-datasource-zookeeper`, `sentinel-logging-slf4j` module has their own separate configuration `java.source.version` with JDK1.7. Suggest unify them into the configuration of the parent pom.xml, which will be more convenient for upgrade in the future.

Reply:
Updated

Reply:
There are still some modules has own `java.source.version` configuration with JDK1.8, `sentinel-demo`,`sentinel-apache-dubbo-adapter`,`sentinel-datasource-redis`, do we need to delete them and keep only one in parent module
JDK1.8 has `LongAdder` class, how about replacing `com.alibaba.csp.sentinel.slots.statistic.base.LongAdder`?

Reply:
Good idea. I've updated, please check.

> The same as Function,Supplier... classes in com.alibaba.csp.sentinel.util.function package.

This may bring breaking changes (as some of them were used as args of exported functions). Maybe we could deprecate them in the next major version.